pkg/fleet-manager/manifests/plugins/testloader.yaml

```diff
@@ -0,0 +1,5 @@
+type: default
+repo: https://flagger.app
+name: loadtester
+version: 0.30.0
+targetNamespace: istio-system
```

Comment:
miss a blank line

Reply:
My pre-push file had a blank line, but the black line miss after push. And I compare other yaml file, find they all miss black lind in the end of file. So I think, it is GITHUB hands it